lib/avalanche/request.ex

```diff
@@ -84,11 +82,17 @@ defmodule Avalanche.Request do
     request_id = get_request_id()
     params = [requestId: request_id]
 
-    options = [base_url: request.url, params: params, auth: {:bearer, request.token}]
+    options = [
+      base_url: request.url,
+      params: params,
+      auth: {:bearer, request.token},
+      retry: [delay: 100, max_retries: 3]
```

Comment:
should we make this configurable?

Reply:
Yes, but not sure we will keep this. It only handles retries for 50x errors. So we will need other retry handlers for things like 429s.